games/c.yaml

```diff
@@ -614,11 +615,11 @@
   content: free
```

Comment:
```suggestion
  content: commercial
```

Reply:
It is free though, unless this line refers to the original game?

Reply:
this is referring to that the game requires original game assets to play - see https://github.com/opengaming/osgameclones/blob/master/schema/games.yaml#L59-L62

Reply:
Ah I missed that.  Updated.